data/json/recipes/recipe_food.json

```diff
@@ -4256,6 +4256,30 @@
   ]
 },{
   "type" : "recipe",
+  "result": "syrup",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_DRINKS",
+  "skill_used": "cooking",
+  "difficulty": 3,
+  "time": 40000,
+  "autolearn": true,
+  "batch_time_factors": [ 60, 3 ],
+  "qualities":[
+    {"id":"BOIL","level":2}
+  ], "tools": [
+    [
+      [ "hotplate", 8 ],
+      [ "toolset", 8 ],
+      [ "fire", -1 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "maple_sap", 35 ]
```

Comment:
The ratio is good. It's simply the charges that refer to different volume. 35 charges of maple sap is 35 x 250 ml and 7 charges of maple syrup is 250 ml. It just means that you can use a single volume (250 ml) of maple syrup 7 times in different recipes which makes sense since you would put that approximative amount (250ml/7 ~= 35 ml) in a recipe.

So

- maple sap: 1 volume is 1 charge = 250 ml
- maple syrup: 1 volume is 7 charges = 250 ml
- take 35 X 1 volume of maple sap and you'll get 1 volume of maple syrup